### DISCUSSION TITLE
Nuxt 3 SSR doesn't work when build on Windows but it works fine when built on ubuntu

### DISCUSSION THREAD

**Discussion by ppunsensor** [nuxt/nuxt#23146] [Kind words]
Hi, Nuxt team
**I've got this error for fresh nuxt3 when I built on Window but no error when built on Ubuntu**
[nuxt] [request error] [unhandled] [500] Cannot find module 'W:\vhosts\took\nuxt3_app\.output\server\node_modules\vue-bundle-renderer\runtime' imported from W:\vhosts\took\nuxt3_app\.output\server\chunks\handlers\renderer.mjs

Step to produces: 
1. `npx nuxi@latest init nuxt3_app`
2. `cd nuxt3_app`
3. `npm install`
4. `npm run build`
5. `npm run start`

***Remark***
My app files store on ubuntu but I develop on windows using SAMBA to map drive

Here is a package.json
```
{
  "name": "nuxt-app",
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare",
    "start": "node .output/server/index.mjs"
  },
  "devDependencies": {
    "@nuxt/devtools": "latest",
    "nuxt": "^3.7.1"
  }
}
```

nuxt.config.ts
```
export default defineNuxtConfig({
  devtools: { enabled: false }
})
```

`node -v`
--v18.13.0

When I request 127.0.0.1:3000 on browser It show error code "500 This page is temporarily unavailable."

I have tried to find many solutions on google but it still doesn't work. Please check this error.

**Comment by CHitenK** [comment]
I've had bugs like this before. 
I used node V22.2.0 to run build.
In the end, there was no such bug;
